ProjectShop/UserApp/views.py

```diff
@@ -87,3 +89,32 @@ def create(self, request):
         except TokenError as error:
             return Response(str(error), status=status.HTTP_202_ACCEPTED)
         return Response(status=status.HTTP_200_OK)
+
+
+class UserViewSet(viewsets.ModelViewSet):
+    serializer_class = UserSerializer
+    queryset = User.objects.all()
+    lookup_field = 'phone_number'
+
+    def patch(self, request):
+        user = User.objects.get(phone_number=request.data.get('phone_number'))
+        if user is None:
+            return Response(status=status.HTTP_400_BAD_REQUEST,
+                            data={'message': 'We haven\'t user with such phone number.'})
```

Comment:
"We don't have a user with such phone number." would be more accurate

Reply:
I've already deleted this methods)